.github/workflows/release.yml

```diff
@@ -144,3 +144,12 @@ jobs:
             ./cl-dag-l1.jar
             ./cl-dag-l1.sha256
             ./mainnet/mainnet-seedlist
+
+      - name: Publish SDK module to Sonatype Central
+        run: sbt tessellation-ci-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
the end of the workflow seemed the most reasonable but I'm happy to move somewhere else if there is a preference